src/main.ts

```diff
@@ -1,14 +1,28 @@
-// The Vue build version to load with the `import` command
-// (runtime-only or standalone) has been set in webpack.base.conf with an alias.
+import 'babel-polyfill';
 import Vue from 'vue';
-import App from '@/App';
+import 'vue-material/dist/vue-material.min.css';
+import {
+  MdToolbar,
+  MdButton,
+  MdIcon,
+} from 'vue-material/dist/components';
+
+import App from '@/App.vue';
```

Comment:
I typically setup an alias for `src` called `@` so we can make absolute imports and webpack will convert to relative imports on build time. This makes refactors much easier because if we move a file, we don't have to change the import paths in those files